src/table/pagination-table/components/head/HeadCell.tsx

```diff
@@ -74,7 +76,13 @@ const HeadCell = ({ column, columnIndex, columnsLength }: HeadCellProps) => {
   const handleOpenMenu = () => setOpen(true);
 
   const newHeadCellRelatedLogic = isNewHeadCellMenuEnabled ? column.colIdx === 0 : false;
-  const tabIndex = newHeadCellRelatedLogic && isInteractionEnabled && areTabStopsEnabled(keyboard) ? 0 : -1;
+  const tabIndex =
+    newHeadCellRelatedLogic &&
+    isFirstHeaderCell(focusedCellCoord) &&
+    isInteractionEnabled &&
+    areTabStopsEnabled(keyboard)
+      ? 0
+      : -1;
```

Comment:
I thought about this for a bit. I think it should be slightly different. So for the different keyboardNavigation cases:
- false: there should be a tabindex = 0 where `focusedCellCoord` is. So we could basically check if `columnIndex` is the same as in `focusedCellCoord`, then add that tabstop
- true: when you press enter on the chart, you should focus the cell that has the `focusedCellCoord` coordinate. On the first render this is [0, 0]. But in that case, we shouldn't add that tabstop, until `keyboard.active` becomes true

So you should do this instead
```suggestion
  const tabIndex =
    newHeadCellRelatedLogic &&
    !keyboard.enabled
    isThisCellFocused && // see comment above
    isInteractionEnabled &&
      ? 0
      : -1;
```